### PR TITLE
Refactor/game stats

### PR DESCRIPTION
**Pull request by javenb022**
## Refactored methods- 
- [x] percentage_home_wins
- [x] percentage_visitor_wins
- [x] percentage_ties
- [x] average_goals_per_game

Built respective tests for methods
Simplecov 100% coverage

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -197,6 +193,13 @@ def fewest_tackles(season)
   end
 
   ### HELPER METHODS ###
+  def percentage(stats)
+    (stats / number_of_games).round(2)
+  end
+  def number_of_games
```
Comment (MylesNottingham) [inline lib/stat_tracker.rb:200]: I need space...

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -166,6 +166,18 @@
   end
 
   ### HELPER METHODS ###
+  describe "#percentage(stats)" do
+    it "gives a float to the nearest 100th percent" do
+      expect(@stat_tracker.percentage(2687)).to eq(0.36)
+      expect(@stat_tracker.percentage(10322)).to eq(1.39)
+      expect(@stat_tracker.percentage(19881)).to eq(2.67)
+    end
+  end
```
Comment (MylesNottingham) [inline spec/stat_tracker_spec.rb:175]: I reallly neeed space!

File: spec/stat_tracker_spec.rb [hunk 3]
```diff
@@ -166,6 +166,18 @@
...
+  describe "#number_of_games" do
+    it "can get the total number of games as a float" do
+      expect(@stat_tracker.number_of_games).to eq(7441.0)
+    end
+  end
```
Comment (MylesNottingham) [inline spec/stat_tracker_spec.rb:181]: srsly

**Review by MylesNottingham — CHANGES_REQUESTED** [review]
Overall great, I'm just feeling a bit crowded.

**Review by javenb022 — COMMENTED** [review]
Made changes per Myles request

**Review by MylesNottingham — APPROVED** [review]

File: lib/stat_tracker.rb [hunk 4]
```diff
@@ -40,27 +40,24 @@ def lowest_total_score
   end
 
   def percentage_home_wins
-    number_of_games = @games.count.to_f
     home_wins = @games.find_all do |game|
       game.home_goals > game.away_goals
     end
-    (home_wins.count / number_of_games).round(2)
+    percentage(home_wins.count)
```
Comment (garrettgregor) [inline lib/stat_tracker.rb:46]: Great use of this helper method!

File: lib/stat_tracker.rb (hunk 1 above)
Comment (garrettgregor) [reply lib/stat_tracker.rb:200]: Just to clarify what Myles is referencing, I think he'se looking for a space at 198.5

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (garrettgregor) [reply spec/stat_tracker_spec.rb:175]: +1

**Review by garrettgregor — CHANGES_REQUESTED** [review]
@javenb022 I agree with @MylesNottingham on the formatting around the methods and test that are referenced in the review. Once you change those let me know and you can merge to get that extra commit!

**Review by garrettgregor — APPROVED** [review]
Changes look great! Space where needed!

**Pull request merged**